### DISCUSSION TITLE
23.3.1 - Dark Theme not installable

### DISCUSSION THREAD

**Discussion by JamesInform** [dbeaver/dbeaver#22277] [Q&A] [answered]
### Description

Using DBeaver on Mac I am not able to install "Darkest Theme" on version 23.3.1.

23.3.0 works without any issues.

### DBeaver Version

23.3.1

**Comment by alesc2000** [comment]
Linux also:

`Cannot complete the install because one or more required items could not be found.
  Software being installed: DevStyle (includes Darkest Dark Theme) 1.11.0.202306151725 (com.genuitec.eclipse.theming.core.feature.feature.group 1.11.0.202306151725)
  Missing requirement: Webclipse's Theming Contributions 1.11.0.202306151725 (com.genuitec.eclipse.theming.ui 1.11.0.202306151725) requires 'osgi.bundle; org.w3c.css.sac 0.0.0' but it could not be found
  Cannot satisfy dependency:
    From: DevStyle (includes Darkest Dark Theme) 1.11.0.202306151725 (com.genuitec.eclipse.theming.core.feature.feature.group 1.11.0.202306151725)
    To: org.eclipse.equinox.p2.iu; com.genuitec.eclipse.theming.ui [1.11.0.202306151725,1.11.0.202306151725]`

**Comment by serge-rider** [comment, marked as answer]
It is an issue in Darkest Dark plugin - it is not fully compatible with Eclipse 2023-12 platform.
Hopefully they will release a fixed version soon.

Workaround: add Eclipse 2023-09 update site to repositories and enable "Contact all update sites" checkbox.
- Got to Install new Software dialog.
- Copy paster `Eclipse 2023-09 - https://download.eclipse.org/releases/2023-09/` into Work with field.
- Click "Add ..."
- Go back and select `Darkest Dark Theme` in Work with.
- Turn on "Contact all update sites during install to find required software" enabled
- Install

**Comment by valsamis-d** [comment]
> It is an issue in Darkest Dark plugin - it is not fully compatible with Eclipse 2023-12 platform. Hopefully they will release a fixed version soon.
> 
> Workaround: add Eclipse 2023-09 update site to repositories and enable "Contact all update sites" checkbox.
> 
>     * Got to Install new Software dialog.
> 
>     * Copy paster `Eclipse 2023-09 - https://download.eclipse.org/releases/2023-09/` into Work with field.
> 
>     * Click "Add ..."
> 
>     * Go back and select `Darkest Dark Theme` in Work with.
> 
>     * Turn on "Contact all update sites during install to find required software" enabled
> 
>     * Install

@serge-rider this worked like a charm :) Thanks!

Just out of curiosity how do I find which version of Eclipse DBeaver Portable uses?

**Comment by Destrolaric** [comment]
@valsamis-d As a rule of thumb, DBeaver is mostly using the latest Eclipse version. There may be a 1-2 week delay between Eclipse and DBeaver release, but it's true in most cases.

**Comment by draos** [comment]
> It is an issue in Darkest Dark plugin - it is not fully compatible with Eclipse 2023-12 platform. Hopefully they will release a fixed version soon.
> 
> Workaround: add Eclipse 2023-09 update site to repositories and enable "Contact all update sites" checkbox.
> 
> [...]

It worked! Thanks.

**Comment by Destrolaric** [comment]
I am glad that worked, I am going to convert the issue to discussion due to it being third-party.

**Comment by mnott** [comment]
Is there actually a way to preserve all settings on update? I'm on a Mac here, and each and every time I do an update, I've to reinstall all plugins and do all settings. It's hyper annoying.

**Comment by natenho** [comment]
The workaround is not working anymore, can anyone confirm that?

**Reply by draos** [reply]
It worked two or three days ago to me. Obviously, I am not willing to check now...  :-D

**Reply by avingochea** [reply]
It just worked for me 10 min ago

**Reply by natenho** [reply]
Ok...I had to set the theme again (in Preferences menu) and it worked. Thanks for the replies ;)
